### PR TITLE
Add authorization token to request header

### PR DESCRIPTION
**Pull request by pyropy**
- Adds authorization API token to JSON fetch requests

### CODE REVIEW

**Comment by pyropy** [comment]
We'd also need to set `API_TOKEN` in the GH secrets.

**Comment by juliangruber** [comment]
> We'd also need to set `API_TOKEN` in the GH secrets.

Did you already do that?

**Review by juliangruber — APPROVED** [review]
code changes lgtm

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ jobs:
 
     - name: Build project
       run: npm run build  # Assumes your project has a build script in package.json
+      env:
+        API_TOKEN: ${{ secrets.API_TOKEN }}
```
Comment (bajtos) [inline .github/workflows/deploy.yml:37]: Kudos for not leaking the API token and using secrets instead! 👏🏻

**Review by bajtos — APPROVED** [review]
👍🏻

**Comment by pyropy** [comment]
> > We'd also need to set `API_TOKEN` in the GH secrets.
> 
> Did you already do that?

I wasn't able to as I don't have permissions

**Comment by juliangruber** [comment]
I added the secret

**Review by juliangruber — APPROVED** [review]

**Pull request merged**